CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
-## 3.0.2 (Unreleased)
+## 3.0.3 (Unreleased)
```

Comment:
Unreleased lines aren't needed anymore. That was part of the HashiCorp tooling but since moving to our own releases we don't need these placeholders.

Reply:
Removed unreleased.